x/emissions/metrics/metrics_util.go

```diff
@@ -0,0 +1,199 @@
+package metrics
```

Comment:
not the biggest fan of adding code we don't use or need. at best, we pay more for audits

ideally we keep and use more of them where it makes sense asap, but perhaps beyond this PR